edgelet/edgelet-core/src/module.rs

```diff
@@ -677,6 +678,7 @@ mod tests {
             ("running", ModuleStatus::Running),
             ("stopped", ModuleStatus::Stopped),
             ("failed", ModuleStatus::Failed),
+            ("dead", ModuleStatus::Dead),
```

Comment:
I don't think EdgeAgent handles other modules getting into the dead state correctly. We need to handle that as well.